dockstore-integration-testing/src/test/java/io/dockstore/client/cli/GeneralIT.java

```diff
@@ -418,6 +419,38 @@ private void verifySourcefileChecksums(final List<Tag> tags) {
         });
     }
 
+    /**
+     * Tests that the language version in WDL tools are set correctly.
+     */
+    @Test
+    public void testWDLToolLanguageVersion() {
+        io.dockstore.openapi.client.ApiClient client = getOpenAPIWebClient(USER_2_USERNAME, testingPostgres);
```

Comment:
As a separate ticket, it might be about time to split out the tests that use swagger and openapi clients into separate classes, like `SwaggerGeneralIT` and `OpenapiGeneralIT` to avoid package names.

Reply:
Created https://ucsc-cgl.atlassian.net/browse/SEAB-5066